src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/patches/PatchSearchPanelViewModel.java

```diff
@@ -166,10 +156,24 @@ public List<PatchListQuickFilter> getQuickFilters() {
         return List.of(openFilter, closedFilter, mergedFilter, archivedFilter);
     }
 
+    @Override
+    public void setList(List list) {
```

Comment:
there's a valid warning from Qodana here, this should be refactored to not use plain `List` without generics